CMakeLists.txt

```diff
@@ -201,6 +201,7 @@ option(AMALGAMATION_BUILD
        "Build from the amalgamation files, rather than from the normal sources."
        FALSE)
 
+option(LINK_EXTENSIONS "Link the extensions into the main libraries." TRUE)
```

Comment:
Could we switch this and add a flag `DISABLE_LINK_EXTENSIONS` that defaults to false instead?

Reply:
Ah sorry didn't realize you commented inline.
What about naming it EXTERNAL_EXTENSIONS?